server/container/guice/guice-common/src/main/java/org/apache/james/GuiceJamesServer.java

```diff
@@ -75,10 +74,6 @@ public void stop() {
         }
     }
 
-    public GuiceServerProbe serverProbe() {
```

Comment:
GuiceJamesServer is not the final application, modules can be overriden/added...
you don't know at this level which probes are binded.